changelog.txt

```diff
@@ -1,6 +1,7 @@
 *** Changelog ***
 
 = 9.7.0 - xxxx-xx-xx =
+* Dev - Makes use of the new product type constants in the whole codebase
```

Comment:
I think it may be worth clarifying the details a bit for the changelog entry, just so it's clearer what we're doing.
```suggestion
* Dev - Use product type constants that were added in WooCommerce 9.7
```